### ISSUE TITLE
Time stretch / pitch shift

### ISSUE THREAD

**Issue by sagamusix** [emoon/HippoPlayer#32]
I don't quite know why but some people make use of this a lot. But it was a really interesting feature of ModPlug Player where you could change pitch and tempo independently from each other (e.g. you could pitch the song up and at the same time halve its speed). Of course this works quite well with note-based formats, and libopenmpt has an interface for doing it. Most other library will probably not support it, but it could be a per-plugin feature/setting.

**Comment by emoon** [comment]
Ah that is a good idea. Maybe they use it for module mixing? :)

**Comment by sagamusix** [comment]
That was in fact another rather unique feature in MPP - you could load and play two modules at the same time, and of course adjust their tempo and pitch to match each other. But I know some people just generally do this not just with modules and also when just playing a single song.

**Comment by emoon** [comment]
Yeah I actually remember that feature now when you mention it. As you say most players won't support it so it would have to be on a per plugin feature setting.
